packages/main/test/pages/Icon_and_theming.html

```diff
@@ -34,9 +34,83 @@
 	<ui5-icon name="tnt-v3/actor"></ui5-icon>
 
 	<br/><br/>
-	<div>BS Icons - single version ("business-suite/add-polygone")</div>
-	<ui5-icon class="samples-margin" name="business-suite/add-polygone"></ui5-icon>
+	<div>BS Icons - single version ("business-suite-v1/add-polygone")</div>
```

Comment:
It's also important to have the "Auto" example without specifying the version as this will be used most of the time.
See the above examples for SAP-Icons and SAP-Icons-TNT. They all follow the pattern: first shows the "Auto" mode and then the versioned APIs -  v2 and v3 for SAPIcons-TNT and v4, v5 for SAPIcons